File: app/src/main/java/com/example/sunkai/heritage/Data/userCommentData.java
package com.example.sunkai.heritage.Data;

import java.io.Serializable;

/**
 * Created by sunkai on 2017-4-24.
 * 此类用于存储用户他发帖的信息
 * 实现了Serializable可以传入至bundle中
 */

public class userCommentData implements Serializable {
    public int id,inListPosition,user_id;
    public String commentTime,commentTitle,commentContent,commentLikeNum,commentReplyNum,userName;
    public byte[] userCommentIamge;
    public byte[] userImage;
    public boolean isUserLike=false;
    public boolean isUserFocusUser=false;
}

File: app/src/main/java/com/example/sunkai/heritage/Data/userCommentData.kt
package com.example.sunkai.heritage.Data

import com.example.sunkai.heritage.ConnectWebService.BaseSettingNew.Companion.SUCCESS
import java.io.Serializable

/**
 * Created by sunkai on 2018/2/21.
 */
class UserCommentData(val id:Int,
                      val commentTime:String,
                      val userName:String,
                      var commentTitle:String,
                      var commentContent:String,
                      val userID:Int,
                      var isLike:String,
                      var isFollow:String,
                      var likeNum:Int,
                      var replyNum:Int,
                      var imageUrl:String):Serializable{
    fun isLike():Boolean{
        return SUCCESS==isLike
    }

    fun isFollow():Boolean{
        return SUCCESS==isFollow
    }
}